### PR TITLE
Support --logspec (glog -vmodule) for finegrained logging

### PR DESCRIPTION
**Pull request by smarterclayton**
Also enable importer logging for server starts.

@liggitt

### CODE REVIEW

File: pkg/cmd/flagtypes/glog.go [hunk 1]
```diff
@@ -15,4 +15,16 @@ func GLog(flags *pflag.FlagSet) {
 		levelPtr := (*int32)(level)
 		flags.Int32Var(levelPtr, "loglevel", 0, "Set the level of log output (0-5)")
 	}
+	if flag := from.Lookup("vmodule"); flag != nil {
+		value := flag.Value
+		flags.Var(pflagValue{value}, "loglevel-spec", "Set per module logging with file|pattern=LEVEL,...")
```
Comment (liggitt) [inline pkg/cmd/flagtypes/glog.go]: logmodule? logspec?

**Comment by smarterclayton** [comment]
[test]

**Comment by liggitt** [comment]
LGTM

**Comment by smarterclayton** [comment]
[merge]

**Pull request merged**